### PR TITLE
Adapt some PSA services to be able to run in a virtualized environment

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

File: JupyterLab/resources/catalog/JupyterLab.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <variable name="NATIVE_SCHEDULER_PARAMS" value=""/>
     <variable name="INSTANCE_NAME" value="jupyterlab-server-$PA_JOB_ID"/>
     <variable name="ENDPOINT_ID" value="jupyterlab-endpoint-$PA_JOB_ID"/>
-    <variable name="PROXYFIED" value="False" model="PA:Boolean"/>
+    <variable name="PROXYFIED" value="True" model="PA:Boolean"/>
```
Comment (mklkun) [inline JupyterLab/resources/catalog/JupyterLab.xml:10]: Why do we have to switch the default value to `True`? Is this mandatory?

Comment (fviale) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: I made this request :-)

It is mandatory in proactive server deployed in https due to the http header strict-transport-security 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security

File: JupyterLab/resources/catalog/Jupyterlab_Service.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <variable name="NATIVE_SCHEDULER" value=""/>
     <variable name="NATIVE_SCHEDULER_PARAMS" value=""/>
     <variable name="ENDPOINT_ID" value="jupyterlab-endpoint-$PA_JOB_ID"/>
-    <variable name="PROXYFIED" value="False" model="PA:Boolean"/>
+    <variable name="PROXYFIED" value="True" model="PA:Boolean"/>
```
Comment (fviale) [inline JupyterLab/resources/catalog/Jupyterlab_Service.xml:9]: can you keep it lowercase ?

File: JupyterLab/resources/catalog/JupyterLab.xml (hunk 1 above)
Comment (fviale) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: @aminelouati There must be a default password configured for JupyterLab (which should be explained in the workflow description)

Comment (mklkun) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: Fair enough. But then a second question, why not changing this default value for all our PSA services?

File: JupyterLab/resources/catalog/Jupyterlab_Service.xml (hunk 2 above)
Comment (mklkun) [reply JupyterLab/resources/catalog/Jupyterlab_Service.xml:9]: This is seen just in the xml file. It was used with capital F/T in all PSA services. I agree that in lowercase it is better but we should also do this change in all PSA service workflows (management workflows will be deleted anyway :'( lol)

File: JupyterLab/resources/catalog/JupyterLab.xml (hunk 1 above)
Comment (fviale) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: It should be done on all PSA service which currently support proxyfied mode.

Comment (aminelouati) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: @fviale OK I will add a default password for JupyterLab explained in the workflow description.
But for info even with an empty password, the jupyterLab web interface is accessible in a proxyfied mode.

Comment (aminelouati) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: I mean we can login with an empty password

Comment (fviale) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: @aminelouati So it displays the form to enter a password and you enter nothing inside ?

Comment (aminelouati) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: Yes exactly. 
Another point, some services such as Knime are not working anymore even with the proxyfied mode. When I opened network tab in the developer tools I got this error
Status Code: 504 Gateway Timeout

Comment (fviale) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: Regarding the password, it is ok to leave it empty, but the description should include this mention.

I don't think work as been done on Knime regarding the proxyfied mode.

Comment (aminelouati) [reply JupyterLab/resources/catalog/JupyterLab.xml:10]: As you said knime and many other services do not support the proxified mode. 
I will create a separate card to update them.

**Pull request merged**